contents/docs/user-guides/groups.md

```diff
@@ -0,0 +1,293 @@
+
+Let's say you have a payment app. You want to track & do analysis on the transaction level, not the user level. You can  create a transaction group type, allowing you to answer questions like: "What is the conversion rate per transaction?"
+
+## Groups vs cohorts
+
+Groups differ from cohorts, a cohort is a essentially a list of users. Whereas a groups represent every entity of a certain type.
+
+Following the company example again, you can have a cohort of users which belong to a certain company (e.g. Company A). But you could create a group type of "companies" which represents all companies and their users.
+
+# What can you do with groups?
+
+## Group Analytics
+You can perform any type of analytics (e.g. Trends, Funnels, Retention, etc.) based on groups rather than users (e.g. Number of companies who signed up in the last 28 days, Number of total transactions in the past 14 days, etc.)
+
+## Feature flags
+You can use groups to roll-out your feature flags more consistently. Instead of giving users in the same organization a different experience, roll out the feature flag to whole organizations.
+
+# How our solution works?
+
+With groups, there's two important concepts to remember. These are Groups and Group Types.
+
+## Groups vs Group Types
+
+The group types determine what kind of groups you have. For example, a `company` is a group type. A `transaction` is a group type. There's a hard limit of 5 group types you can create, so think carefully about what group types you want.
+
+A group is a specific instance of a group type. For example, if `company` is a group type, then `PostHog`, `Tesla`, `IBM`, `SpaceX` are the groups. You can have unlimited groups.
+
+As another example, if `transaction` is a group type, then each `transactionID` is a group.
+
+## Defining groups
+
+Groups are at the event level. If everything is setup correctly, then events that come into PostHog have a group associated with them, which is determined by you. See below for how to set this up.
+
+Thus, there's no new events corresponding to groups: it's simply the events that already exist, aggregated by group values.
+
+A popular misconception is that groups are defined per user. This might make sense if the group type is a company, since a company is a set of users, but it quickly breaks down when you're looking at a transaction group type: here a user can do multiple transactions. Thus, remember that groups are defined on events.
+
+Another important concept to remember here is that a single event belongs to only a single group per group type. For example, say you have a `transaction occured` event. And you have 3 group types: `transaction`, `company`, `pineapple_type`.
+
+Then, this event can belong to at most one group per group type. So, for this event, you can have `transaction = 123456789`, `company = PostHog`, `pineapple_type = British`.
+
+You can't have something like: `transaction = [123456789, 12345678910]`. This is not possible.
```

Comment:
:thinking: is a negative example a good idea? Will someone copy-pasta it/misunderstand?

Reply:
Added a warning to make it clearer